.github/workflows/validate-pr.yml

```diff
@@ -254,7 +254,7 @@ jobs:
     - name: Setup GraalVM environment
       uses: olafurpg/setup-scala@v10
```

Comment:
This action is deprecated. I think it'd be better to migrate to https://github.com/graalvm/setup-graalvm